src/engine/defaults.ts

```diff
@@ -10,5 +10,6 @@ export const defaults = {
   cname: undefined,
   dryRun: false,
   remote: 'origin',
+  ghPagesUrl: '',
```

Comment:
probably the prop should just be named `url`? @JohannesHoppe / @fmalcher  what do you think?

Reply:
First of all: Thanks for the PR! It's a great start..🤗

Regarding this part:
I don't think this should be an option at all.  There is no need to set this via options, it should be only shown in the output.  Such a change would also make the overall PR smaller!

Reply:
I agree with your point: `There is no need to set this via options`.
Anyway, I'll make another patch to apply the feedback.